src/CheckoutSdk/Sessions/Completion/CompletionInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+
+namespace Checkout.Sessions.Completion
+{
+    public abstract class CompletionInfo
+    {
+        [JsonProperty(PropertyName = "type")]
+        protected readonly CompletionInfoType Type;
```

Comment:
Maybe "[JsonProperty(PropertyName = "type")]" it isn't necessary

Reply:
Without this tag (JsonProperty) the protected member will not be present in the json. I should have added this decoration